scripts/vagrant-linux-priv-config.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+export DEBIAN_FRONTEND=noninteractive
+
+# Update and ensure we have apt-add-repository
+apt-get update
+apt-get install -y software-properties-common
+
+# Add i386 architecture (for libraries)
+dpkg --add-architecture i386
+
+# Add a Golang PPA
+sudo add-apt-repository ppa:gophers/archive
+
+# Add the Docker repository
+apt-key adv \
+	--keyserver hkp://p80.pool.sks-keyservers.net:80 \
+	--recv-keys 9DC858229FC7DD38854AE2D88D81803C0EBFCD88
+add-apt-repository \
+	"deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+	$(lsb_release -cs) \
+	stable"
+
+# Update with i386, Go and Docker
+apt-get update
+
+# Install Core build utilities for Linux
+apt-get install -y \
+	build-essential \
+	git-core \
```

Comment:
Plain old `git` finally works for the VCS these days.

Reply:
That's what I thought too, I copied this from the existing Linux script though so left it as it was. I'll change this to use `git` directly.